crates/pecos-qec/src/rot_surface/patch_layout.rs

```diff
@@ -0,0 +1,89 @@
+use std::collections::HashMap;
+
+// TODO: Maybe... Add Plaquettes with a center, vertices, edges (boundary), and types. (Edges from center to vertices?)
+//  maybe vertices => plaquettes...
+//  maybe overall boundaries
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct RotSurfLayout {
+    pub id2coords: HashMap<usize, (usize, usize)>,
+    pub vertices: Vec<usize>,
+    pub centers: Vec<usize>,
+}
+
+#[allow(clippy::module_name_repetitions)]
+#[must_use]
+pub fn gen_patch_layout(height: usize, width: usize) -> RotSurfLayout {
```

Comment:
Nit: the way the mutable references are passed to `new_node` seems like it could be refactored but probably not worth the hassle unless you start getting into borrow checker issues

Reply:
I think there is some Rust code that just needs to be removed for this release that I left in that isn't fully baked yet. I'll clean it up and wait to introduce it later on. Need another dev branch for my dev branch.

Reply:
Basically, ignore `pecos-qec` for now.